src/NServiceBus.Metrics.ServiceControl/ReportingOptions.cs

```diff
@@ -1,10 +1,13 @@
 ﻿namespace NServiceBus.Metrics.ServiceControl
 {
+    using System;
+
     class ReportingOptions
     {
         internal string ServiceControlMetricsAddress;
         internal string EndpointInstanceIdOverride;
-
+        public TimeSpan TimeToBeReceived { get; set; } = TimeSpan.FromDays(7);
```

Comment:
@Scooletz Would it make sense to log this information? The destination and TTBR that is used?

Reply:
That would be on the endpoint side. What would be the value of it?

Reply:
If people provide logfiles when they complain about metrics never arriving in ServiceControl? I doubt it'll ever get raised as an issue, but it might happen. Then it's immediate proof.
Unless @ramonsmits has another idea.

Reply:
If we followed this approach, the logging queue should be also logged. I see no value in it (highly unlikely to be the issue), but if you find it a must, I can add it.

Reply:
@Scooletz Yes, the destination too. I think it always makes sense to log configured values as long as they do not contain sensitive information. It is not a must to add but especially as we are promoting code API here it is hidden what the configured value is.